### ISSUE TITLE
Void Siphon doesn't generate Void Seeds when powered by Impetus Diffuser

### ISSUE THREAD

**Issue by janipez** [TheCodex6824/ThaumicAugmentation#367]
As per title, it rapidly drains the provided Impetus from a Diffuser, but doesn't generate any Void Seeds.

Also tested with Impetus provided to a Diffuser by an Impetus Matrix, so I can rapidly see it draining, but there are still no Seeds being formed.

**Comment by LincT** [comment]
More details might help as I've successfully formed the seeds from the diffuser. 
- it's a little slow as I'm balancing my rift to not drain it completely.

Couple questions:
- How big is the rift that's powering it?
- How many impetus cells are in the matrix to give an idea of impetus
- you're on the latest version of the mod?
- Are there any other thaumcraft related mods installed?

for reference, my rift outputs a power level of 14 and when it tips over to 15 is when I pull impetus from it into the siphon
I also keep a stability generator rigged to fire as needed.
![image](https://github.com/TheCodex6824/ThaumicAugmentation/assets/11671267/0c48369b-d900-4519-813e-1ed466928a96)
![image](https://github.com/TheCodex6824/ThaumicAugmentation/assets/11671267/3f3c6123-4367-405c-a907-991bd6359259)

**Comment by janipez** [comment]
- Tried all rift sizes ranging from 1 to 15, now I just set up the Siphon near the rift alongside the Drainer, hooked up on the same signals.
![image](https://github.com/TheCodex6824/ThaumicAugmentation/assets/17101663/df71fab7-cbc8-4528-856d-407e309ebf44)

- Poured the entire Matrix containing 4 cells into it, no Seeds formed.
![image](https://github.com/TheCodex6824/ThaumicAugmentation/assets/17101663/83593da3-c765-4562-a9a8-868b71a34002)


- Yes, latest version of the mod.
- Inventory Scanning and JEI.

**Comment by TheCodex6824** [comment]
I'll look into this. Are you playing singleplayer or on a server? I remember I had to do some special things to make the created void seeds in the siphon sync properly, but maybe something broke.

**Comment by TheCodex6824** [comment]
I tested this a bit today and things seem to be working with a basic test setup. One thing you could try: go into a creative world and connect the creative impetus source (red cube thing) to a diffuser and see if that works. I did identify an issue where very small amounts of impetus (literally only 1 or 2) getting sent to the diffuser will do nothing, but if you were using large rifts I don't think that case would come up.

**Comment by LincT** [comment]
I tested further in the apr 11th, 2023 release version (the issue is still legit, the workaround also legit)

For anyone coming to this wanting a workaround while waiting for the next production/release version:

- Use a redstone relay to filter for the signal from the rift monitor (I set mine to 15)
- Pipe the output to a pulse extender (I'm using a double comparator extender)
- use the pulse extender to turn off a redstone torch (or otherwise disable a redstone signal locking the void siphon)

tested by 
- breaking and replacing the siphon to reset it
- pulse extender using 2 comparators, 1 solid block, 3 redstone
- smelted 8 stacks netherwart in a void smelter (~645 vitium)
- fed into a rift outputting a signal of 14
yielded 1 seed

repeating the experiment with 4 comparators for the delay, and similar netherwart (~325 vitium) yielded similar results.

Conclusion:
It seems it's enough to delay for a few ticks as long as there's a suitably strong rift outputting charged impetus on demand to the diffuser. Be careful not to set the pulse delay so long as to completely drain the rift.


(tried to clip out blocks from other mods that do not affect this use case)

    ![image](https://github.com/TheCodex6824/ThaumicAugmentation/assets/11671267/d2462c9f-1e82-4d24-9560-d3020c0ea2d5)
![image](https://github.com/TheCodex6824/ThaumicAugmentation/assets/11671267/13784ee9-7fd3-46de-b98f-f989d1668e16)

**Comment by LincT** [comment]
I think I found something pertinent.
On my multiplayer world I wasn't getting void seeds either. 
Comparing the two worlds I realized I changed one thing.
- I wasn't passing impetus through a matrix, simply straight from the rift when I was getting seed.
- Removing the impetus matrix from the line, I was able to get a seed off less than 2 jars
This might help pinpoint why the issue 
- we might be on an edge case where the matrix is outputting a lot of small increments of impetus instead of a larger increment 
- This would tie back to the issue with void siphons not working with too small an amount drained at a time.
> TL;DR don't have a matrix in front of your siphon
